lucet-runtime/lucet-runtime-internals/src/future.rs

```diff
@@ -195,17 +189,22 @@ impl InstanceHandle {
                             ))
                         }
                     }
+                    InternalRunResult::BoundExpired => Ok(Bounce::BoundExpired),
                 }
-            })?;
+            }?;
             match bounce {
```

Comment:
FWIW now that there's no `wrap_blocking` I think it should be fine to do all this processing of `Bounce` directly inline above instead of having it separated down here.

Reply:
Indeed, that's much cleaner; thanks.